crypto/fipsmodule/self_check/self_check.c

```diff
@@ -1198,10 +1198,15 @@ int BORINGSSL_self_test(void) {
       // When requested to run self tests, also run the lazy tests.
       !boringssl_self_test_rsa() ||
       !boringssl_self_test_ecc() ||
-      !boringssl_self_test_ffdh()) {
+      !boringssl_self_test_ffdh() ||
+      !boringssl_self_test_sha256() ||
+      !boringssl_self_test_hmac_sha256()) {
```

Comment:
why add these?

Reply:
It made my testing simpler, so all the algorithms behave the same. Now if the constructor or if a customer calls BORINGSSL_self_test they get the same results.